instructors.html

```diff
@@ -27,21 +27,45 @@
         <div class="span10 offset1">
           <h1 class="title">Version Control with Git</h1>
           <h2 class="subtitle">Instructor's Guide</h2>
-<h2 id="legend">Legend</h2>
-<p>Wolfman and Dracula will work together in one project.</p>
-<h2 id="overall">Overall</h2>
+<p>Version control might be the most important topic we teach, but Git is definitely the most complicated tool. However, GitHub presently dominates the open software forge landscape, so we have to help novices learn just enough Git to feel they can and should learn more on their own.</p>
```

Comment:
Agreed that this paragraph comes close to conflating git and github. It's also true that git presently dominates the open software landscape, so why not just say that?